src/assets/custom/css/_sass/_compass-teaser.scss

```diff
@@ -23,3 +23,14 @@
   color: white;
   margin-bottom: 25px;
 }
+
+.blog-post-compass-teaser {
+  background-color: $coral;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background: url("#{$site-base-url}/assets/custom/img/softmod/bg-compass-cta.jpg");
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Maybe all the common areas could be styled together:

```.compass-teaser,
.blog-post-compass-teaser { ```

And then we can have a separate entry for any changes to try and reduce the repetition. We can have a call and I can explain further if need be.

Reply:
I hear ya. All sorted